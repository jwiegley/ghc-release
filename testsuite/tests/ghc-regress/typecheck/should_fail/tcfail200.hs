{-# OPTIONS -fglasgow-exts #-}

module ShouldFail where

f = let x = ( 1#, 'c' ) in x
