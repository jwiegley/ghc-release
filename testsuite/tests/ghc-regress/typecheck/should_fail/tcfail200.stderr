
tcfail200.hs:5:14:
    Couldn't match kind `#' against `*'
    When matching the kinds of `GHC.Prim.Int# :: #' and `t :: *'
      Expected type: t
      Inferred type: GHC.Prim.Int#
    In the expression: 1#
